conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -5,6 +5,11 @@
 # changes to this script, consider a proposal to conda-smithy so that other feedstocks can also
 # benefit from the improvement.
 
+{{ group_functions_bash }}
```

Comment:
Can we move this to a helper script and source it here?

Reply:
I don't know if I got it right, but see last commit.